svc.html

```diff
@@ -0,0 +1,347 @@
+      [[!WEBRTC]], including the <dfn>RTCPeerConnection</dfn> object (defined in Section 4.4),
+      the <dfn>RTCRtpParameters</dfn> dictionary (defined in Section 5.2.1),
+      the <dfn>RTCRtpSendParameters</dfn> dictionary (defined in Section 5.2.2),
+      the <dfn>RTCRtpReceiveParameters</dfn> dictionary (defined in Section 5.2.3),
+      the <dfn>RTCRtpCodingParameters</dfn> dictionary (defined in Section 5.2.4),
+      the <dfn>RTCRtpEncodingParameters</dfn> dictionary (defined in Section 5.2.6),
+      the <dfn>RTCRtpTransceiver</dfn> object including
+      its <dfn>[[\Sender]]</dfn> and <dfn>[[\Receiver]]</dfn> internal slots (defined
+      in Section 5.4), the <dfn>RTCRtpSender</dfn> object, including its
+      <dfn>[[\SendEncodings]]</dfn> and <dfn>[[\LastReturnedParameters]]</dfn> internal slots (defined in Section 5.2),
+      and the <dfn>RTCRtpReceiver</dfn> object (defined in Section 5.3).</p>
+  </section>
+  <section id="operational-model">
+    <h2>Scope and Operational model</h2>
+    <p>This specification extends the WebRTC specification [[!WEBRTC]] to
+    enable user agents to support scalable video coding (SVC).</p>
+    <p>For Scalable Video Coding (SVC), the terms single-session transmission
+    (<dfn>SST</dfn>) and multi-session transmission (<dfn>MST</dfn>) are defined in
+    [[RFC6190]]. This specification only supports <a>SST</a> but not <a>MST</a>. The
+    term Single Real-time transport protocol stream Single Transport (<dfn>SRST</dfn>),
+    defined in [[RFC7656]] Section 3.7, refers to Scalable Video Coding
+    (<dfn>SVC</dfn>) implementations that transmit all layers within
+    a single transport, using a single Real-time Transport Protocol (RTP) stream
+    and synchronization source (SSRC). The term Multiple RTP stream Single Transport
+    (<dfn>MRST</dfn>), also defined in [[RFC7656]] Section 3.7, refers to
+    implementations that transmit all layers within a single transport, using
+    multiple RTP streams with a distinct SSRC for each layer. This specification
+    only supports <a>SRST</a> transport.</p>
+    <p>Within this specification it is assumed that scalable video coding operation is primarily
+    controlled utilizing the <code>addTransceiver</code> method of the <code><a>RTCPeerConnection</a></code>
+    and the <code>setParameters</code> method of the <code><a>RTCRtpSender</a></code> object.</p>
+    <p>The <code>addTransceiver</code> method establishes the <dfn>SVC envelope</dfn> which includes
+    the maximum number of scalable video coding layers that can be sent, as well as the ordering of the <code>encodings</code>.
+    While characteristics of individual scalable video coding layers can be modified using the <code>setParameters</code>
+    method, the <a>SVC envelope</a> cannot be changed.  One of the implications of this model is that
+    the <code>addTrack</code> method is not suitable for use with scalable video coding since this method
```

Comment:
Tangent: Huh, I guess you're right. Wouldn't it make sense for addTrack to initialize with a default encoding parameters dictionary? I would not expect setParameters() to fail for non-simulcast cases if the sender was created by addTrack and not addTransceiver.